htdocs/manage/settings/index.bml

```diff
@@ -32,8 +32,16 @@ body<=
     my $authas;
     my $u;
 
-    if ($remote) {
-        $authas = $GET{authas} || $remote->user;
+    $authas = $GET{authas} || $remote->user if $remote;
+
+    my $can_view_other = ( $remote && $remote->has_priv( "canview", "subscriptions" ) );
+
+    if ( $can_view_other && $GET{cat} && $GET{cat} eq 'notifications' ) {
+        # impersonation mode - check for $u->user ne $authas
+        $u = LJ::load_user( $GET{user} ) if $GET{user};
```

Comment:
For code comprehension ability, I think it's nicer to see "if they want to view another user, and they can view another user... then load another user" rather than "if they can view another user, load another user... if they specified one".